internal/util/BasicProviders.go

```diff
@@ -38,15 +37,14 @@ func GetLogger() *zap.SugaredLogger {
 }
 
 type LogConfig struct {
-	Level int `env:"LOG_LEVEL" envDefault:"0"`
+	Level int `env:"LOG_LEVEL" envDefault:"0"` // default info
 }
 
-func init() {
+func InitLogger() *zap.SugaredLogger{
 	cfg := &LogConfig{}
 	err := env.Parse(cfg)
 	if err != nil {
-		fmt.Println(err)
-		return
+		panic("failed to parse logger env config: " + err.Error())
```

Comment:
can we handle error and throw here instead of panic

Reply:
done